src/AspNetCore/Src/aspnetcoreconfig.cxx

```diff
@@ -144,13 +150,19 @@ ASPNETCORE_CONFIG::Populate(
     }
 
     pAdminManager = g_pHttpServer->GetAdminManager();
-
     hr = strSiteConfigPath.Copy(pHttpContext->GetApplication()->GetAppConfigPath());
     if (FAILED(hr))
     {
         goto Finished;
     }
 
+    BSTR test = SysAllocString(pHttpContext->GetApplication()->GetApplicationPhysicalPath());
```

Comment:
why do you need this BSTR test?

Reply:
Poor naming and most likely this should be a STRU. We need the full application path to run it with dotnet.exe.